[BUG] Cannot find module 'puppeteer-core/lib/Browser'

Trying to execute README example, the following error occurred. 

## Environment
* `chrome-aws-lambda` Version: 3.1.1
* `puppeteer` / `puppeteer-core` Version: 5.1.0
* OS: Linux 
* Node.js Version: 12.x 
* Lambda / GCF Runtime: nodejs12.x

## Expected Behavior
Should have returned example.com content.

## Current Behavior
Following error occurs:
```
Lambda execution failed with status 200 due to customer function error:  Cannot find module 'puppeteer-core/lib/Browser'
Require stack:
- /var/task/node_modules/chrome-aws-lambda/source/puppeteer/lib/Browser.js
- /var/task/node_modules/chrome-aws-lambda/source/index.js
- /var/task/app.js
```
No Browser module expored in the following path: 'puppeteer-core/lib/Browser'

## Steps to Reproduce
Lambda functions:  memory is 1024M, timeout is 3000ms.
```js
const chromium = require('chrome-aws-lambda');

exports.handler = async (event, context, callback) => {
  let result = null;
  let browser = null;

  try {
    browser = await chromium.puppeteer.launch({
      args: chromium.args,
      defaultViewport: chromium.defaultViewport,
      executablePath: await chromium.executablePath,
      headless: chromium.headless,
      ignoreHTTPSErrors: true,
    });

    let page = await browser.newPage();

    await page.goto(event.url || 'https://example.com');

    result = await page.title();
  } catch (error) {
    return callback(error);
  } finally {
    if (browser !== null) {
      await browser.close();
    }
  }

  return callback(null, result);
};
```

--- a/source/puppeteer/lib/Browser.js
+++ b/source/puppeteer/lib/Browser.js
@@ -3,7 +3,7 @@ let Super = null;
 try {
   Super = require('puppeteer/lib/cjs/puppeteer/common/Browser').BrowserContext;
 } catch (error) {
-  Super = require('puppeteer-core/lib/cjs/common/Browser').BrowserContext;
+  Super = require('puppeteer-core/lib/cjs/puppeteer/common/Browser').BrowserContext;
 }
 
 let newPage = Super.prototype.newPage;

--- a/source/puppeteer/lib/FrameManager.js
+++ b/source/puppeteer/lib/FrameManager.js
@@ -3,7 +3,7 @@ let Super = null;
 try {
   Super = require('puppeteer/lib/cjs/puppeteer/common/FrameManager').Frame;
 } catch (error) {
-  Super = require('puppeteer-core/lib/cjs/common/FrameManager').Frame;
+  Super = require('puppeteer-core/lib/cjs/puppeteer/common/FrameManager').Frame;
 }
 
 /**

--- a/source/puppeteer/lib/Page.js
+++ b/source/puppeteer/lib/Page.js
@@ -3,7 +3,7 @@ let Super = null;
 try {
   Super = require('puppeteer/lib/cjs/puppeteer/common/Page').Page;
 } catch (error) {
-  Super = require('puppeteer-core/lib/cjs/common/Page').Page;
+  Super = require('puppeteer-core/lib/cjs/puppeteer/common/Page').Page;
 }
 
 /**

